laravel/app/Http/Controllers/StockController.php

```diff
@@ -19,6 +19,19 @@ public function order()
         return view('order', $data);
     }
 
+    public function ordersindex(Request $request)
```

Comment:
複数の英単語を繋げる時は `_` で繋げるようにしましょう（スネークケース）
```suggestion
    public function order_index(Request $request)
```